reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -60,8 +67,7 @@ def __init__(
         self.remove_seen = remove_seen
 
         self.similarity_type = similarity_type
-        # convert to seconds
-        self.time_decay_half_life = time_decay_coefficient * 24 * 60 * 60
+        self.time_decay_half_life = time_decay_coefficient * 24 * 60 * 60  # convert to seconds
         self.time_decay_flag = timedecay_formula
         self.time_now = time_now
         self.threshold = threshold
```

Comment:
I think this can be removed: `self.model_str = "sar_ref"` it comes from the old development airship, and we are not using it anymore in the other algos

Reply:
ok, sounds good